lib/ReactTags.js

```diff
@@ -384,8 +384,8 @@ ReactTags.propTypes = {
   delimiters: PropTypes.array,
   autofocus: PropTypes.bool,
   inline: PropTypes.bool,
-  handleDelete: PropTypes.func.isRequired,
-  handleAddition: PropTypes.func.isRequired,
+  handleDelete: PropTypes.func,
```

Comment:
can you move proptypes and default props inside the class and make it static ?

Reply:
I certainly can, but that seems out of scope for this PR. if you'd like me to, I can do it, but I can also put up a separate PR that makes the mechanical change to separate it from the functionality change. up to you

Reply:
yes that works. You can give a separate PR for the change.